other-platforms.md

```diff
@@ -19,3 +19,12 @@ The Lottie community has build Lottie bindings for the following platforms:
 * [Windows](https://github.com/windows-toolkit/Lottie-Windows)
 * [Qt](https://blog.qt.io/blog/2019/03/08/announcing-qtlottie/)
 * [Skia](https://skia.org/user/modules/skottie)
+
+# MIT Appinventor
```

Comment:
Can you add this as a line below the Python wrapper above because that is what this is? The link should include whatever information somebody might need such as what you have included below. That does not need to be here.